src/test/java/nurseybook/model/task/TaskTest.java

```diff
@@ -177,18 +180,6 @@ void isTaskOverdue() {
         assertTrue(keithInsulin.isTaskOverdue()); // status: isOverdue = "true"
     }
 
-    @Test
-    void markTaskDone() {
```

Comment:
Are these 2 methods covered in other test cases?

Reply:
Same reasoning as above, will be changed in future implementation by @xiongjya.